control_panel_api/tests/test_pagination.py

```diff
@@ -0,0 +1,29 @@
+from model_mommy import mommy
+from rest_framework.reverse import reverse
+from rest_framework.test import APITestCase
+
+from control_panel_api.tests.test_views import AuthenticatedClientMixin
+
+
+class TestPagination(AuthenticatedClientMixin, APITestCase):
+    def test_pagination(self):
+        mommy.make('control_panel_api.App', 120)  # 120 exceeds the 100 default
```

Comment:
Rather than explain this with comments, it would be self-documenting to have a variable `default_page_size = 100` and then `mommy.make('...App', default_page_size + 20)`

Reply:
🎖